convert/threejs/fromColladaToThreejs.test.js

```diff
@@ -0,0 +1,26 @@
+import '@jsxcad/algorithm-cgal';
+
+import { boot } from '@jsxcad/sys';
+import { fromColladaToThreejs } from './fromColladaToThreejs.js';
+import { fromThreejsToGeometry } from './fromThreejsToGeometry.js';
+import { prepareForSerialization } from '@jsxcad/geometry';
+import { readFileSync } from 'fs';
+import test from 'ava';
+
+test.beforeEach(async (t) => {
+  await boot();
+});
+
+test('Simple import', async (t) => {
+  const data = readFileSync('duck_triangles.dae');
+  const threejs = await fromColladaToThreejs(data);
+  const geometry = JSON.parse(
+    JSON.stringify(
+      prepareForSerialization(await fromThreejsToGeometry(threejs))
+    )
+  );
+  const expectedGeometry = JSON.parse(
+    readFileSync('duck_triangles.json', 'utf8')
+  );
+  t.deepEqual(geometry, expectedGeometry);
```

Comment:
Sometimes I forget how good the testing setup is. I think that was a critical decision getting the testing in early and maintaining it. We used to have regressions all the time, but they haven't been a problem in a while.